### ISSUE TITLE
Member access error

### ISSUE THREAD

**Issue by sefo** [palantir/tslint#3217]
#### Bug report

- __TSLint version__: 5.7.0
- __TypeScript version__: 2.5.2
- __Running TSLint via__: VSCode

#### TypeScript code being linted

```ts
export class MenuNavLargeComponent extends MenuNavComponent {
  constructor() {
    super();
  }
}
```
```ts
export abstract class MenuNavComponent {
  menu: string[];
}
```
```html
<div *ngFor="let m of menu"></div>
```

#### Actual behavior

Error: The property "menu" that you're trying to access does not exist in the class declaration.

#### Expected behavior
No error.

In the template if I replace menu by this['menu'] the error goes away but it doesn't seem like a clean solution.

**Comment by ajafff** [comment]
That's not an error originating from a TSLint core rule.
I'm pretty certain you want to file this issue over there instead: https://github.com/mgechev/codelyzer
